msdk-io/msdk-io-mzml2/src/main/java/io/github/msdk/io/mzml2/util/MzMLIntensityPeaksDecoder.java

```diff
@@ -208,16 +228,21 @@ public static DecodedData decode(byte[] bytesIn, int lengthIn, Integer precision
           float asDouble;
 
           for (int i = 0; i < numPoints; i++) {
-            offset = i * chunkSize;
-
-            asLong = ((long) (decoded[offset] & 0xFF)) // zero shift
-                | ((long) (decoded[offset + 1] & 0xFF) << 8)
-                | ((long) (decoded[offset + 2] & 0xFF) << 16)
-                | ((long) (decoded[offset + 3] & 0xFF) << 24)
-                | ((long) (decoded[offset + 4] & 0xFF) << 32)
-                | ((long) (decoded[offset + 5] & 0xFF) << 40)
-                | ((long) (decoded[offset + 6] & 0xFF) << 48)
-                | ((long) (decoded[offset + 7] & 0xFF) << 56);
+            // offset = i * chunkSize;
+
+            // asLong = ((long) (decoded[offset] & 0xFF)) // zero shift
+            // | ((long) (decoded[offset + 1] & 0xFF) << 8)
+            // | ((long) (decoded[offset + 2] & 0xFF) << 16)
+            // | ((long) (decoded[offset + 3] & 0xFF) << 24)
+            // | ((long) (decoded[offset + 4] & 0xFF) << 32)
+            // | ((long) (decoded[offset + 5] & 0xFF) << 40)
+            // | ((long) (decoded[offset + 6] & 0xFF) << 48)
+            // | ((long) (decoded[offset + 7] & 0xFF) << 56);
```

Comment:
Any reason you aren't deleting this code instead of commenting out? If you have good git messages, you can always find this code after deleting it...